roles/step_acme_cert/templates/step-post-renew-hook.sh.j2

```diff
@@ -0,0 +1,12 @@
+#!{{ step_acme_cert_post_renewal_shell }}
+####### added by ansible: maxhoesel.smallstep.step_acme_cert - changes will be overwritten #######
+set -eu
+export STEP_CLI="{{ step_cli_executable_absolute.stdout }}"
+export CERT_FILE="{{ step_acme_cert_certfile_full.path }}"
+export KEY_FILE="{{ step_acme_cert_keyfile_full.path }}"
```

Comment:
I don't know how I feel about these. I get that they're convenient shorthands and that we kind of need them (since we are using `step_acme_key/certfile_*full*` internally, so we can't just tell people to use `step_came_cert_certfile.path` or whatever), but i'm wondering whether there's a way to avoid adding these. Then again, I guess having them doesn't hurt either ...